Makefile

```diff
@@ -23,13 +23,11 @@ KONG_BUILD_TOOLS ?= `grep KONG_BUILD_TOOLS .requirements | awk -F"=" '{print $$2
 KONG_VERSION ?= `cat kong-*.rockspec | grep tag | awk '{print $$3}' | sed 's/"//g'`
 
 setup-release:
-	if cd kong-build-tools; \
-	then git pull; \
-	else git clone https://github.com/Kong/kong-build-tools.git; fi
+	-rm -rf kong-build-tools; \
+	git clone https://github.com/Kong/kong-build-tools.git; fi
```

Comment:
We can do `git clone -b $(KONG_BUILD_TOOLS) https://github.com/Kong/kong-build-tools.git; fi` to save the `git reset --hard` later.

Reply:
that won't work if we set `KONG_BUILD_TOOLS` to a sha